db/schema.rb

```diff
@@ -761,7 +761,15 @@
     t.boolean  "immigrant"
     t.datetime "uwfs_training_date"
     t.boolean  "clearinghouse_record_found"
+    t.boolean  "international_student"
+    t.string   "intended_major"
+    t.integer  "transfer_student"
+    t.integer  "residency_status"
+    t.string   "home_state"
+    t.integer  "housing_status"
     t.boolean  "parental_consent_on_file"
+    t.boolean  "pending_participant"
+    t.string   "intake_form_signate"
```

Comment:
I seem to always get these unwanted fields whenever I migrate my database. I ran migrations for these fields in a different branch few months ago, but I no longer needed these. One of them is a typo. 
Any suggestion to prevent this?